srml/executive/src/lib.rs

```diff
@@ -125,9 +126,11 @@ impl<
 > ExecuteBlock<Block> for Executive<System, Block, Context, Payment, UnsignedValidator, AllModules>
 where
 	Block::Extrinsic: Checkable<Context> + Codec,
-	<Block::Extrinsic as Checkable<Context>>::Checked: Applyable<Index=System::Index, AccountId=System::AccountId>,
+	<Block::Extrinsic as Checkable<Context>>::Checked:
```

Comment:
We use `WeighableCall` trait for `Extrinsic`, which seems a bit misleading - can we please rename it to just `Weighable`?